app/templates/rentals.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <h2>Welcome!</h2>
+  <p>
+    We hope you find exactly what you're looking for in a place to stay.
```

Comment:
Simply removed the second line. I think it looks fine unless anyone has a good suggestion on generic text.